tests/test_crash_reports.py

```diff
@@ -95,6 +97,8 @@ def test_that_top_crashers_reports_links_work(self, mozwebqa, product):
             csp.wait_for_page_to_load()
 
     @pytest.mark.nondestructive
+    @pytest.mark.xfail("'allizom.org' in config.getvalue('base_url')",
+                       reason="S3 bucket is populating with crash data")
```

Comment:
I'm not sure I understand this reason. Is there a related bug we could reference with more details?